src/main/resources/application.properties

```diff
@@ -1,9 +1,9 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/kodilla_project?serverTimezone=Europe/Warsaw&useSSL=False

```

Comment:
Jaka jest przyczyna zmian w tym pliku?

Reply:
przyczyną zmian były błędy pojawiające się podczas pierwszego uruchomienia projektu
"Failed to load driver class com.mysql.cj.jdbc.Driver" 
natomiast została usunięta linia z
"spring.jpa.hibernate.naming.strategy=org.hibernate.cfg.ImprovedNamingStrategy"
prawdopodobnie w trakcie konfliktu przy próbie git pull została omyłkowo usunięta